src/index.js

```diff
@@ -0,0 +1,104 @@
+import express from 'express';
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+
+ app.use(express.json());
+
+
+const database = {
+
+		entries: [{
+				title: "heading", message: "message"
+		}],
+
+		users: [{
+				firstname: "first", lastname: "last", email: "email", password: "password"
+		}],
+
+};
+
+
+app.post('/api/v1/entries', (req, res) =>{
+	
+	const { title, message } = req.body;
+	if (title === ' ' || message === ' ' ) {
+		res.status(422).json('Please fill fields');
+	} 
+	 else if (title && message) {
+		database.entries.push(req.body);
+		res.status(201).json(database.entries);
+	} 
+	else {
+		res.status(400).json('Bad Request');
```

Comment:
This is a wrong status code, I believe you want to catch a server error. 500 should be appropriate.

Reply:
Change this wherever you are repeating yourself.

Reply:
I have updated the server error to 500